Lombiq.NodeJs.Extensions/scripts/clean-assets.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @summary A script to clean asset files; expects configuration in assets-to-copy.json or package.json (assetsToCopy).
+ * @description This script removes assets previously copied using copy-assets.js.
+ */
+const path = require('path');
+const util = require('util');
+const rimraf = util.promisify(require('rimraf'));
+const getAssetsConfig = require('./get-assets-config');
+
+const verbose = true;
+const filePattern = '**/*';
+
+// Change to consuming project's directory.
+process.chdir('..');
+process.chdir('..');
+
+function log(message) {
+    if (verbose) process.stdout.write(message + '\n');
+}
+
+async function deleteFiles(assetsConfig) {
+    log('Executing deleteFiles ...');
+
+    return Promise.all(assetsConfig
+        .map((assetsGroup) => {
+            log(`Cleaning ${assetsGroup.target}`);
+
+            return rimraf(path.join(assetsGroup.target, filePattern));
+        }));
+}
+
+(async function main() {
+    log('Executing clean-assets.js ...');
```

Comment:
Why do you add spaces in front of the punctuation?
If you want to delineate the file name, rather wrap it in quotes.